### PR TITLE
Doc changes to address frequently asked questions

### PR DESCRIPTION
**Pull request by nategraf**
### Description

This PR makes doc and minor CLI changes to address frequently asked questions and issues found on Discord. Additionally I fixed some broken links that I found with [remarkjs](https://github.com/remarkjs/remark-validate-links)

### CODE REVIEW

**Review by lucasege — APPROVED** [review]

File: packages/cli/src/commands/election/activate.ts [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+import { cli } from 'cli-ux'
 import { BaseCommand } from '../../base'
 import { newCheckBuilder } from '../../utils/checks'
 import { displaySendTx } from '../../utils/cli'
 import { Flags } from '../../utils/command'
 
 export default class ElectionVote extends BaseCommand {
-  static description = 'Activate pending votes in validator elections to begin earning rewards'
+  static description =
+    'Activate pending votes in validator elections to begin earning rewards. It is required to activate your pending votes at some point after the end of the epoch in which they were made.'
```
Comment (asaj) [inline packages/cli/src/commands/election/activate.ts]:
```suggestion
    'Activate pending votes in validator elections to begin earning rewards. To earn rewards as a voter, it is required to activate your pending votes at some point after the end of the epoch in which they were made.'
```

File: packages/docs/command-line-interface/election.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ description: Participate in and view the state of Validator Elections
 
 ### Activate
 
-Activate pending votes in validator elections to begin earning rewards
+Activate pending votes in validator elections to begin earning rewards. It is required to activate your pending votes at some point after the end of the epoch in which they were made.
```
Comment (asaj) [inline packages/docs/command-line-interface/election.md]:
```suggestion
Activate pending votes in validator elections to begin earning rewards. To earn rewards as a voter, is required to activate your pending votes at some point after the end of the epoch in which they were made.
```

File: packages/docs/getting-started/running-a-validator.md [hunk 3]
```diff
@@ -10,15 +10,15 @@ While other Validator Groups will exist on the Celo Networks, the fastest way to
...
+Additionally, Validators are expected to run an [Attestation Service](https://github.com/celo-org/celo-monorepo/tree/master/packages/attestation-service) as part of the [lightweight identity protocol](../celo-codebase/protocol/identity), to provide attestations that allow users to map their phone number to a Celo address.
 
 You can find more details about Celo mission and why to become a Validator [in our Medium article](https://medium.com/celohq/calling-all-chefs-become-a-celo-validator-c75d1c2909aa).
 
-## Prerequisites
+## Register for the Stake Off
 
-### Register for the Stake Off
+Participation in The Great Celo Stake Off is subject to these [Terms and Conditions](https://docs.google.com/document/d/1b5SzeRbq60nx50NeezAEMpwLkaBDQ9hjZc0QAh4Mbdk/). If you agree to those, register online via an [online form](https://docs.google.com/forms/d/e/1FAIpQLSfbn5hTJ4UIWpN92-o2qMTUB0UnrFsL0fm97XqGe4VhhN_r5A/viewform). **Once the C-Labs team receive your registration, they will send you instructions to get fauceted funds to run a Validator on the Baklava testnet.** Do this first.
```
Comment (asaj) [inline packages/docs/getting-started/running-a-validator.md]:
```suggestion
Participation in The Great Celo Stake Off is subject to these [Terms and Conditions](https://docs.google.com/document/d/1b5SzeRbq60nx50NeezAEMpwLkaBDQ9hjZc0QAh4Mbdk/). If you agree to those, register online via an [online form](https://docs.google.com/forms/d/e/1FAIpQLSfbn5hTJ4UIWpN92-o2qMTUB0UnrFsL0fm97XqGe4VhhN_r5A/viewform). **Once the C-Labs team receives your registration, they will send you instructions to get fauceted funds to run a Validator on the Baklava testnet.** Do this first.
```

**Review by asaj — APPROVED** [review]
Very nice!

**Comment by timmoreton** [comment]
Looks like you need to autogen the cli docs

File: packages/docs/command-line-interface/election.md [hunk 4]
```diff
@@ -6,15 +6,15 @@ description: Participate in and view the state of Validator Elections
 
 ### Activate
 
-Activate pending votes in validator elections to begin earning rewards
+Activate pending votes in validator elections to begin earning rewards. To earn rewards as a voter, is required to activate your pending votes at some point after the end of the epoch in which they were made.
```
Comment (asaj) [inline packages/docs/command-line-interface/election.md]:
```suggestion
Activate pending votes in validator elections to begin earning rewards. To earn rewards as a voter, it is required to activate your pending votes at some point after the end of the epoch in which they were made.
```

**Pull request merged**